docs/README.md

```diff
@@ -1,19 +1,20 @@
-[//]: # Licensed to the Apache Software Foundation (ASF) under one
-[//]: # or more contributor license agreements.  See the NOTICE file
-[//]: # distributed with this work for additional information
-[//]: # regarding copyright ownership.  The ASF licenses this file
-[//]: # to you under the Apache License, Version 2.0 (the
-[//]: # "License"); you may not use this file except in compliance
-[//]: # with the License.  You may obtain a copy of the License at
-[//]: #
-[//]: # http://www.apache.org/licenses/LICENSE-2.0
-[//]: #
-[//]: # Unless required by applicable law or agreed to in writing,
-[//]: # software distributed under the License is distributed on an
-[//]: # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-[//]: # KIND, either express or implied.  See the License for the
-[//]: # specific language governing permissions and limitations
-[//]: # under the License.
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
```

Comment:
The `#` actually makes this into giant header text when viewing the readme on github. The `[//]: # ` doesn't work as expected either. If you want to sink a little time into figuring out how to make this a proper markdown comment block, that'd be appreciated. Both of the aforementioned treatments don't work as intended.

Reply:
Weird! I did this via the Github merge conflict UI, hmm I'l fix locally!

Reply:
I see it's the `<!--` syntax now... that was causing other errors for me (couldn't run the build script), but maybe you'll have better luck?